entrypoint.sh

```diff
@@ -159,6 +169,34 @@ function get_files_matching_filetype() {
 	echo "${matching_files[@]}"
 }
 
+function load_filetype_array() {
+	local val=$1
+	declare -a types
+
+	val=$(echo "$val" | jq -r '.[]')
+	while IFS= read -r filetype; do
+		types+=("$filetype")
+	done <<<"$val"
+
+	echo "${types[@]}"
+}
+
+function has_autofix() {
+	local lint_name=$1
+
+	while IFS= read -r line; do
+		name=$(echo "$line" | jq -r ".name")
+		if [[ "$name" == "$lint_name" ]]; then
+			if echo "$line" | jq -e '.autofix' > /dev/null; then
+				# If linter has an autofix exit successfully
```

Comment:
We're mixing how we use return.  In other functions return 0 means we didn't find one like [here](https://github.com/SeisoLLC/goat/blob/a967dfc01eeabf4a3cfea5a611fc448d1b353276/entrypoint.sh#L94-L108).  Whatever we choose, let's stay consistent